helper-functions.ts

```diff
@@ -0,0 +1,99 @@
+import {ByteArray, BigInt} from "./index";
+
+/**
+ * Takes 2 ByteArrays and concatenates them
+ * @param a - 1st ByteArray
+ * @param b - 2nd ByteArray
+ * @returns A concatenated ByteArray
+ */
+function concat(a: ByteArray, b: ByteArray): ByteArray {
+  let out = new Uint8Array(a.length + b.length);
+  for (let i = 0; i < a.length; i++) {
+    out[i] = a[i];
+  }
+  for (let j = 0; j < b.length; j++) {
+    out[a.length + j] = b[j];
+  }
+  return out as ByteArray;
+}
+
+/**
+ * Used for parseCSV() below
+ */
+enum CSVState {
+  BETWEEN = 0,
+  UNQUOTED_VALUE = 1,
+  QUOTED_VALUE = 2,
+}
+
+/**
+ * Parses a CSV string into an array of strings.
+ * @param csv CSV string.
+ * @returns Array of strings.
+ */
+function parseCSV(csv: string): Array<string> {
+  let values = new Array<string>()
+  let valueStart = 0
+  let state = CSVState.BETWEEN
+
+  for (let i: i32 = 0; i < csv.length; i++) {
+    if (state == CSVState.BETWEEN) {
+      if (csv[i] != ',') {
+        if (csv[i] == '"') {
+          state = CSVState.QUOTED_VALUE
+          valueStart = i + 1
+        } else {
+          state = CSVState.UNQUOTED_VALUE
+          valueStart = i
+        }
+      }
+    } else if (state == CSVState.UNQUOTED_VALUE) {
+      if (csv[i] == ',') {
+        values.push(csv.substr(valueStart, i - valueStart))
+        state = CSVState.BETWEEN
+      }
+    } else if (state == CSVState.QUOTED_VALUE) {
+      if (csv[i] == '"') {
+        values.push(csv.substr(valueStart, i - valueStart))
+        state = CSVState.BETWEEN
+      }
+    }
+  }
+
+  return values
+}
+
+/**
+ * Check if two BigInts are equal by comparing each character in the string
+ * @param a - 1st BigInt to compare
+ * @param b - 2nd BigInt to compare
+ * @returns   True if they are equal, false if they are not
+ */
+export function equals(a: BigInt, b: BigInt): boolean {
```

Comment:
This one we should copy paste into index.ts as the `@operator('==')` for BigInt

Reply:
agreed, i will do this now in this PR

Reply:
awesome!